cmd/tsbs_generate_queries/uses/devops/common.go

```diff
@@ -157,41 +161,46 @@ func GetMaxAllLabel(dbName string, nHosts int) string {
 	return fmt.Sprintf("%s max of all CPU metrics, random %4d hosts, random %s by 1h", dbName, nHosts, MaxAllDuration)
 }
 
-func getRandomHosts(scale, nHosts int) []string {
-	if nHosts < 1 {
-		fatal("number of hosts cannot be < 1; got %d", nHosts)
+// getRandomHosts returns a subset of numHosts hostnames of a permutation of hostnames,
+// numbered from 0 to totalHostsScale. Like 5 out of 30
+// Ex.: host_12, host_7, host_25 for numHosts=3 and totalHostsScale=30
+func getRandomHosts(numHosts int, totalHostsScale int) []string {
```

Comment:
I think `totalHosts` suffices, shorter and conveys the same thing.

Reply:
Totally agreed, this would be cleaner.
Fixed.